components/ToDo/index.js

```diff
@@ -0,0 +1,55 @@
+import useLocalStorageState from "use-local-storage-state";
+import {
+  InputField,
+  ToDoBody,
+  AddButton,
+  DeleteButtonToDo,
+  ToDoBackground,
+  ToDoP,
+  InputPosition,
+} from "./index.styled";
+
+export default function ToDo() {
+  const [todos, setTodos] = useLocalStorageState("todos", { defaultValue: [] });
+  const [task, setTask] = useLocalStorageState("task", { defaultValue: "" });
+
+  function addTodo() {
+    if (task !== "") {
+      setTodos([...todos, task]);
+      setTask("");
+    }
+  }
+
+  function deleteTodo(index) {
+    const newTodos = todos.filter((_, i) => i !== index);
+    setTodos(newTodos);
+  }
```

Comment:
It would be less error prone to have some sort of id. The `index` is not a good choice for deleting elements from your dynamic list.